thinkhazard/static/less/report.less

```diff
@@ -252,6 +252,21 @@ body {
   }
 }
 
+footer {
+  display: table;
+  white-space: nowrap;
+  &:before, &:after {
+    border-top: 5px solid #e6e6e6;
+    content: '';
+    display: table-cell;
+    position: relative;
+    top:0.5em ;
+    width: 45%;
+  }
+  &:before { right: 1.5%; }
+  &:after { left: 1.5%; }
```

Comment:
Those two lines should be removed.
If you look at the mockup, the line is aligned with the content above.